charms/kfp-schedwf/src/components/pebble_component.py

```diff
@@ -45,8 +46,9 @@ def get_layer(self) -> Layer:
                         "override": "replace",
                         "summary": "scheduled workflow controller service",
                         "startup": "enabled",
-                        "command": "/bin/controller --logtostderr=true"
-                        " --namespace={self.namespace}",
+                        "command": f"/bin/controller --logtostderr=true"
+                        f" --namespace={self.namespace}"
```

Comment:
This will be changed by https://github.com/canonical/kfp-operators/pull/529 so let's not include it in this PR.

```suggestion

```

Reply:
Good catch, pushed a commit for this (that sets namespace to `""` value)